engine/sql/065_workflow_node_run_job_infos.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS "workflow_node_run_job_info" (
+    id BIGSERIAL PRIMARY KEY,
+    workflow_node_run_job_id BIGINT,
+    workflow_node_run_id BIGINT,
+    created TIMESTAMP WITH TIME ZONE,
+    spawninfos JSONB
+);
+
+SELECT create_foreign_key_idx_cascade('FK_WORKFLOW_NODE_RUN_INFOS_WORKFLOW_NODE_RUN_JOB', 'workflow_node_run_job_info', 'workflow_node_run_job', 'workflow_node_run_job_id', 'id');
```

Comment:
Should the foreign key be on workflow_node_run_id, because the workflow_node_run_job would be drop ?

Reply:
the cascade is on workflow_node_run_job. When we delete the job from workflow_node_run_job, it will delete the infos too.
Spawninfos are stored into workflow_run after end of job (as before).

Reply:
okay, i did not understood the impact on sync, but that's fine